editor.planx.uk/src/pages/Team/index.tsx

```diff
@@ -123,25 +82,43 @@ const FlowItem: React.FC<FlowItemProps> = ({
   teamSlug,
   refreshFlows,
 }) => {
+  const [isArchiving, setIsArchiving] = useState<boolean>(false);
```

Comment:
nit: `isArchiving` reads to me like an active process (i.e. the operation is running). What this really is referring to is if the dialog is open. Maybe something more explicit like `isArchiveDialogOpen` would be more appropriate here.